jekyll/_ccie/overview.md

```diff
@@ -50,12 +69,17 @@ Our Builder Boxes handle running your builds, and store no state themselves. Eac
 | Services Box                     | all traffic / all ports | Internal Communication                                         |
 | Builder Boxes (including itself) | all traffic / all ports | Internal Communication                                         |
 
-#### GitHub
----
+CircleCI Enterprise uses GitHub Enterprise credentials for
+authentication which, in turn, may use LDAP, SAML, or SSH for access. The following table describes the ports used on machines running GitHub to communicate with the Services and Builder instances.
```

Comment:
I have some concern that this will read as if we require those auth mechanisms. Suggest we change to "which, in turn..." to ", so it will inherit whatever integration work you have done there to your central SSO infrastructure." (if you don't make this change for some reason, the inclusion of SSH is a bit misleading in that we don't have that as a way to authenticate to CircleCI itself)